lib/compat/wordpress-6.5/fonts/class-wp-font-utils.php

```diff
@@ -158,7 +158,7 @@ public static function sanitize_from_schema( $tree, $schema ) {
 				}
 
 				$is_value_array  = is_array( $value );
-				$is_schema_array = is_array( $schema[ $key ] );
+				$is_schema_array = is_array( $schema[ $key ] ) && ! is_callable( $schema[ $key ] );
```

Comment:
Adding this check allows using a class instance method, like `array( $this, 'sanitization_method' )`.